web/blog/apache-arrow-as-platform-for-security-data-engineering/index.md

```diff
@@ -59,8 +59,7 @@ we started using Arrow as optional dependency as additional column-oriented
 encoding. We abstracted a batch of data encoding-independent behind a "table
```

Comment:
I can't comment on a submodule bump, so here's a comment on an unrelated line: This PR should not be changing the CAF submodule from 0.18.7 back down to 0.18.6.

Reply:
Thanks for the 🦅 👀 !